package error_handling_with_exceptions.task9.myException;

public class ThreeException extends ArithmeticException{

}
